sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -74,6 +74,60 @@ export interface DeleteConfigurationSettingOptions extends HttpOnlyIfUnchangedFi
 export interface DeleteConfigurationSettingResponse extends SyncTokenHeaderField, HttpResponseFields, HttpResponseField<SyncTokenHeaderField> {
 }
 
+// @public
+export interface FeatureFlag extends FeatureFlagParam, ConfigurationSetting {
+}
+
+// @public
+export const featureFlagContentType = "application/vnd.microsoft.appconfig.ff+json;charset=utf-8";
+
+// @public
+export interface FeatureFlagParam extends ConfigurationSettingParam {
+    conditions: {
+        clientFilters: (FeatureFlagTargetingClientFilter | FeatureFlagTimeWindowClientFilter | FeatureFlagPercentageClientFilter | Record<string, unknown>)[];
+    };
+    description?: string;
+    enabled: boolean;
+}
+
+// @public
+export interface FeatureFlagPercentageClientFilter {
+    name: "Microsoft.Percentage";
+    parameters: {
+        value: number;
+    };
+}
+
+// @public
+export const featureFlagPrefix = ".appconfig.featureflag/";
+
+// @public
+export interface FeatureFlagTargetingClientFilter {
+    name: "Microsoft.Targeting";
+    parameters: {
+        audience: {
+            users: string[];
+            groups: {
+                name: string;
+                rolloutPercentage: number;
+            }[];
+            defaultRolloutPercentage: number;
+        };
+    };
+}
+
+// @public
+export interface FeatureFlagTimeWindowClientFilter {
+    name: "Microsoft.TimeWindow";
+    parameters: {
+        start: string;
```

Comment:
I'm curious if we'll be able to pull together some decent jsdoc strings (perhaps from the portal, at least)

Reply:
Added some docs for start and end